src/app/search/basic-search/basic-search.component.scss

```diff
@@ -7,3 +7,8 @@ textarea:focus {
 .form-field {
   display: inherit;
 }
+
+.search-position {
+  vertical-align: middle;
```

Comment:
need explanation on why this search bar has different styling than everything else on the site

Reply:
The search bar previously had a class of "w-100 mb-3" so I added the margin bottom in the styling and the alignment for the button.

Reply:
so i think the implementation is going backwards. .form-field should be removed, this .search-position should be removed. the w-100 and mb-3 should be removed. And a w-100 should be added to the form field.  and mat-label should be added.

This should result in a smaller form field which is more inline with the mockups and this also has the advantage of having little to no css.